### ISSUE TITLE
merc bomb caused server crash

### ISSUE THREAD

**Issue by bretonium** [yavl/teeworlds-infclassR#46]
```
(gdb) where
#0  0x000055c24fea7271 in CPlayer::IsInfected (this=0x0) at src/game/server/player.cpp:828
#1  0x000055c24feaa43a in CGameContext::CreateExplosion (this=0x7ff985a11010, Pos=..., Owner=7, Weapon=0, NoDamage=false, TakeDamageMode=0, MercBomb=true)
    at src/game/server/gamecontext.cpp:411
#2  0x000055c24fee4430 in CGrowingExplosion::Tick (this=0x55c251451a00) at src/game/server/entities/growingexplosion.cpp:170
#3  0x000055c24fea0ea3 in CGameWorld::Tick (this=0x7ff985a14178) at src/game/server/gameworld.cpp:236
#4  0x000055c24fead939 in CGameContext::OnTick (this=0x7ff985a11010) at src/game/server/gamecontext.cpp:1137
#5  0x000055c24fe7b610 in CServer::Run (this=0x7ff980754010) at src/engine/server/server.cpp:2161
#6  0x000055c24fe7e3ff in main (argc=1, argv=0x7fff7854d398) at src/engine/server/server.cpp:2772
```
This is my line 411 of `gamecontext.cpp`: https://github.com/bretonium/my-infclass-server/blob/33522cbdbf91c66f81da549a31e10eb0c8b359e5/src/game/server/gamecontext.cpp#L411 . As we can see, `m_apPlayers[Owner]` is null for some reason.

**Comment by bretonium** [comment]
The fix would be to check it on not being null i guess

**Comment by bretonium** [comment]
Fixed by pr #45
